manager/container.go

```diff
@@ -331,20 +335,28 @@ func (cd *containerData) GetProcessList(cadvisorContainer string, inHostNamespac
 		if err != nil {
 			return nil, fmt.Errorf("invalid virtual size %q: %v", fields[7], err)
 		}
+		cmd := fields[10]
 		// convert to bytes
 		vs *= 1024
-		psr, err := strconv.Atoi(fields[11])
+		fieldNo := 11
+		var psr int
+		psr, err = strconv.Atoi(fields[fieldNo])
 		if err != nil {
-			return nil, fmt.Errorf("invalid pid %q: %v", fields[1], err)
+			fieldNo++
+			psr, err = strconv.Atoi(fields[fieldNo])
+			if err != nil {
+				return nil, fmt.Errorf("invalid pid %q: %v", fields[1], err)
+			}
+			cmd += fmt.Sprintf(" %s", fields[11])
```

Comment:
why is this necessary?

Reply:
Because I have forgotten that `strings.Join()` exists :)